pkgs/development/tools/buf/default.nix

```diff
@@ -51,6 +54,14 @@ buildGoModule rec {
       cp "$GOPATH/bin/$FILE" "$out/bin/"
     done
 
+    # Completions
+    installShellCompletion --cmd buf --bash <($out/bin/buf bash-completion) \
+      --fish <($out/bin/buf fish-completion) --zsh <($out/bin/buf zsh-completion)
+
+    # Man Pages
+    mkdir man && $GOPATH/bin/buf manpages man
```

Comment:
We use `$out` for completion above but `$GOPATH` here, I would prefer `$GOPATH` for both.